angular-ngrx-scss/src/app/home/home.component.ts

```diff
@@ -9,11 +9,13 @@ import { fetchUserData } from '../state/user/user.actions';
   styleUrls: ['./home.component.scss'],
 })
 export class HomeComponent implements OnInit {
-  user$ = this.store.select(selectUserLoginName);
+  user$ = this.store.select(selectAuthUserName);
 
   constructor(private store: Store) {}
 
   ngOnInit() {
-    this.store.dispatch(fetchUserData());
+    this.user$.subscribe((user) => {
+      this.store.dispatch(fetchUserData({ username: user }));
+    });
```

Comment:
Oh good catch. The subscription is only here because we need that name from the `user$` selector and this was the simplest way I could figure to pass it to the store effect it's already dispatching. 

Updated this to destroy the call when it's done.